app/scripts/controller-init/messengers/index.ts

```diff
@@ -58,12 +64,4 @@ export const CONTROLLER_MESSENGERS = {
     getMessenger: getTransactionControllerMessenger,
     getInitMessenger: getTransactionControllerInitMessenger,
   },
-  MultichainBalancesController: {
-    getMessenger: getMultichainBalancesControllerMessenger,
-    getInitMessenger: getMultichainBalancesControllerInitMessenger,
-  },
-  MultichainTransactionsController: {
-    getMessenger: getMultichainTransactionsControllerMessenger,
-    getInitMessenger: getMultichainTransactionsControllerInitMessenger,
-  },
```

Comment:
We don't need an init messenger, this is only used for legacy controllers that uses to pass hooks to the constructor.